docs/CaliptraIntegrationSpecification.md

```diff
@@ -599,18 +599,26 @@ The following table describes SoC integration requirements.
 | Mailbox                          | SoC shall provide an access path between the mailbox and the application CPU complex on SoCs with such complexes (for example, Host CPUs and Smart NICs). See the [Sender Protocol](#sender-protocol) section for details about error conditions.             | Statement of conformance | Required for Project Kirkland and TDISP TSM       |
 | Fuses                            | SoC shall burn non-field fuses during manufacturing. Required vs. optional fuses are listed in the architectural specification.                                                                                                           | Test on silicon          | Required for UDS threat model                     |
 | Fuses                            | SoC shall expose an interface for burning field fuses. Protection of this interface is the SoC vendor’s responsibility.                                                                                                                   | Test on silicon          | Required for Field Entropy                        |
 | Fuses                            | SoC shall write fuse registers and fuse done via immutable logic or ROM code.                                                                                                                                                             | Statement of conformance | Required for Caliptra threat model                |
+| Fuses                            | SOC shall expose an API for programming Field Entropy as described in the architecture documentation. SOC shall ensure that Field Entropy can only be programmed via this API and shall explicitly prohibit burning of discrete Field entropy bits and re-burning of already burned Field Entropy entries. | Test on silicon          | Required for Field Entropy                        |
+| Fuses                            | SOC shall ensure that any debug read paths for fuses are disabled in PRODUCTION lifecycle state.| Test on silicon          | Required for Field Entropy                        |
+| Fuses                            | SOC shall ensure that UDS_SEED and Field Entropy supplied to Caliptra come directly from OTP fuses and there are no debug paths to inject new values.| Statement of conformance          | Required for Caliptra Threat Model                        |
+| Fuses                            | SOC shall add integrity checks for Caliptra fuses as per SOC policy. | Statement of conformance | Reliability |
+| Fuses                            | SOC should apply shielding/obfuscation measures to protect fuse macro. | Statement of conformance | Required for Caliptra Threat Model |
```

Comment:
I intentionally kept this as a recommendation rather than a requirement because there is no simple way to shield fuse macro. Integrators might be able to deploy some techniques that point some extent of shielding but it will be difficult to say that the fuse macro is properly shielded.